### ISSUE TITLE
Misleading docblock or incompatible implementation of ConsoleIO

### ISSUE THREAD

**Issue by weirdan** [composer/composer#7199]
`IOInterface::ask()` docblock says it [could be called with array](https://github.com/composer/composer/blob/4d8b9be5b6a0bda0a3d24c0a6ff89d7ccb26f715/src/Composer/IO/IOInterface.php#L106) as a `$question`. ConsoleIO [passes this argument](https://github.com/composer/composer/blob/4d8b9be5b6a0bda0a3d24c0a6ff89d7ccb26f715/src/Composer/IO/ConsoleIO.php#L239) to Symfony\Component\Console\Question\Question::__construct(), but that has [that argument typed as string](https://github.com/symfony/console/blob/9a1625df3f2d51cb73916822e150855a7b22bdba/Question/Question.php#L37) since [about 5 months ago](https://github.com/symfony/console/commit/ff2b2c5f4dedbf573d179d9e9e6254f3b3987026#diff-c1e4372c220f920fc34a19dddde639d6L37).

Not sure if anything inside composer uses that form of call (with array), but external projects like `zend-expressive-skeleton` do, and this makes them incompatible with composer when it uses `symfony/console:^4` (`symfony/console:^3` is fine though).

**Comment by Seldaek** [comment]
Right, looks like it should only accept strings.. I'll fix the docblock to match symfony
